tests/runtime/engine/parser.py

```diff
@@ -95,6 +98,8 @@ def __read_test_struct(test, test_suite):
                 for e in line.split():
                     k, v = e.split('=')
                     env[k]=v
+            elif item_name == 'ARCH':
+                arch = line.split("|")
```

Comment:
Maybe strip whitespace here too?
```suggestion
                arch = [x.strip() for x in line.split("|")]
```

Reply:
ok. Will change it.

Reply:
Added. Thanks